README.md

```diff
@@ -35,10 +35,21 @@ All commands are run from the root of the project, from a terminal:
 | `npm run astro ...`    | Run CLI commands like `astro add`, `astro check` |
 | `npm run astro --help` | Get help using the Astro CLI                     |
 
+## Instructions on how to run your own server
+The archive file contains a dist/ folder that needs to be placed into a copy of this cloned repo. Then you can serve it using grunt serve. On a production site, you should to use these environment variables:
```

Comment:
Ahh, I see... what happened, sorry for not being clear in the issue - but this section should be added to the README in the group-income repo, not the groupincome.org repo. Can you send in a PR there?

For this PR we just need the change below to link to the group-income repo.

(Also, can you add an empty line after the heading too?)